### ISSUE TITLE
REPL: Support easier way of specifying full amount for post_pending_transfer

### ISSUE THREAD

**Issue by tKe** [tigerbeetle/tigerbeetle#2357]
It seems the REPL doesn't support (that I can see) a way to specify `amount_max` on when creating a `post_pending_transfer` transfer - the only way to post the full pending amount currently seems to be:
```bash
tb "create_transfers id=24002 pending_id=24000 flags=post_pending_transfer amount=340282366920938463463374607431768211455;"
```

Attempting to use `amount=-0` results in:
> Expected value after equal sign in key-value pair. e.g. `id=1`.

Ideally either `amount=-0` (documented) or `amount=max` should be supported in the REPL, if only for the `amount` field.

**Comment by tKe** [comment]
as an aside this will also impact `exercises/024_two_phase_transfers.sh` in the tigerlings

**Comment by batiati** [comment]
Thanks @tKe!
Supporting `-0` is a great suggestion!

**Comment by tKe** [comment]
I suggested `-0` primarily as it mirrors the tigerbeetle test syntax, however it may be worth considering supporting  `amount=amount_max` for consistency with the client constants as ultimately that's how most consumers will end up thinking of it.

**Comment by batiati** [comment]
Closed by #2358
